t/t7502-commit-porcelain.sh

```diff
@@ -262,13 +262,57 @@ test_expect_success 'cleanup commit messages (scissors option,-F,-e, scissors on
 
 	echo >>negative &&
 	cat >text <<-\EOF &&
+
 	# ------------------------ >8 ------------------------
 	to be removed
 	EOF
 	git commit --cleanup=scissors -e -F text -a --allow-empty-message &&
 	git cat-file -p HEAD >raw &&
 	sed -e "1,/^\$/d" raw >actual &&
-	test_must_be_empty actual
+	test_must_be_empty actual 
+'
+
+test_expect_success 'cleanup commit messages (scissors option,-F,-e, CR/LF line endings)' '
+
+	echo >>negative &&
+	test_config core.editor unix2dos &&
+	cat >text <<-\EOF &&
+
+	# to be kept
+
+	   # ------------------------ >8 ------------------------
```

Comment:
I am not sure that this works as you intended: as far as I remember, _all_ leading whitespace is removed in `<<-\EOF` constructs.

Reply:
Respectfully, whitespace *was* preserved when I added a `cp text /tmp/text &&` to the test. 

If you don't feel positive about this I'm happy to make changes to my tests, should the value in the [existing test](https://github.com/git-for-windows/git/blob/a98b073ae2b993892641bb1eb6ce707cc673d9ad/t/t7502-commit-porcelain.sh#L241) also be updated?

Reply:
Okay, so the existing tests _already_ used a not-quite-readable construct...

But given that the `printf` (or `test_write_lines`) method combined with a variable called `scissors` relates the _intention_ of the code much better, wouldn't you agree that we should use that, instead of copied-but-not-quite-excellent code?

Reply:
> wouldn't you agree that we should use that, instead of copied-but-not-quite-excellent code?

Of course, I'm just trying to not drift of of my lane. I switched my tests to `test_write_lines` and I agree this is much better.

I also tweaked the language in the commit messages for clarity.